content/theme/11ty-fundamenty.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Fundamenty"
+github: https://github.com/creasoft-dev/fundamenty
+demo: https://creasoft-dev.github.io/fundamenty/
+author: Young-Suk Ahn Park
+date: 2020-07-24
+github_branch: master 
+ssg:
+  - Eleventy`
```

Comment:
Looks like a typo at the end, need to remove the `

Reply:
Thanks for the catch.
Commit with the fix was pushed.